src/engine/enginemaster.cpp

```diff
@@ -726,67 +760,77 @@ void EngineMaster::process(const int iBufferSize) {
         // Update VU meter (it does not return anything). Needs to be here so that
         // master balance and talkover is reflected in the VU meter.
         if (m_pVumeter != nullptr) {
-            m_pVumeter->process(m_pMaster, m_iBufferSize);
+            m_pVumeter->process(m_pMaster, iBufferSize);
         }
     }
 
     if (m_pMasterMonoMixdown->toBool()) {
-        SampleUtil::mixStereoToMono(m_pMaster, m_iBufferSize);
+        SampleUtil::mixStereoToMono(m_pMaster, iBufferSize);
     }
 
     if (masterEnabled) {
-        m_pMasterDelay->process(m_pMaster, m_iBufferSize);
+        m_pMasterDelay->process(m_pMaster, iBufferSize);
     } else {
-        SampleUtil::clear(m_pMaster, m_iBufferSize);
+        SampleUtil::clear(m_pMaster, iBufferSize);
     }
     if (headphoneEnabled) {
-        m_pHeadDelay->process(m_pHead, m_iBufferSize);
+        m_pHeadDelay->process(m_pHead, iBufferSize);
     }
     if (boothEnabled) {
-        m_pBoothDelay->process(m_pBooth, m_iBufferSize);
+        m_pBoothDelay->process(m_pBooth, iBufferSize);
     }
 
     // We're close to the end of the callback. Wake up the engine worker
     // scheduler so that it runs the workers.
     m_pWorkerScheduler->runWorkers();
 }
 
-void EngineMaster::applyMasterEffects() {
+void EngineMaster::applyMasterEffects(int iBufferSize) {
     // Apply master effects
     if (m_pEngineEffectsManager) {
         GroupFeatureState masterFeatures;
         masterFeatures.has_gain = true;
         masterFeatures.gain = m_pMasterGain->get();
-        m_pEngineEffectsManager->processPostFaderInPlace(m_masterHandle.handle(),
-                                                         m_masterHandle.handle(),
-                                                         m_pMaster,
-                                                         m_iBufferSize, m_iSampleRate,
-                                                         masterFeatures);
+        m_pEngineEffectsManager->processPostFaderInPlace(
+                m_masterHandle.handle(),
+                m_masterHandle.handle(),
+                m_pMaster,
+                iBufferSize,
+                m_iSampleRate,
+                masterFeatures);
     }
 }
 
-void EngineMaster::processHeadphones(const CSAMPLE_GAIN masterMixGainInHeadphones) {
+void EngineMaster::processHeadphones(
+        const CSAMPLE_GAIN masterMixGainInHeadphones,
+        int iBufferSize) {
```

Comment:
IMO `iBufferSize` should be `std::size_t` but you probably prefer `SINT`...

Reply:
I have started to change this, but this changes spreads like cancer because int iBufferSize is used everywhere > 1k occurrences. I would like to keep this PR small.